src/Microsoft.Framework.Runtime.Interfaces/project.json

```diff
@@ -1,7 +1,10 @@
 {
-    "version": "1.0.0-*",
-    "description": "ASP.NET 5 assembly neutral interfaces to take advantage of capabilities exposed by the runtime. The interfaces are embedded resources that will become part of your ASP.NET 5 application during build.",
-    "compilationOptions": { "warningsAsErrors": true },
+	"version": "1.0.0-*",
+	"description": "ASP.NET 5 assembly neutral interfaces to take advantage of capabilities exposed by the runtime. The interfaces are embedded resources that will become part of your ASP.NET 5 application during build.",
```

Comment:
The description still says ANI... needs to change? Anywhere else it needs to change?